code/modules/library/lib_machines.dm

```diff
@@ -299,6 +297,19 @@ var/global/list/datum/cachedbook/cachedbooks // List of our cached book datums
 	popup.set_title_image(user.browse_rsc_icon(src.icon, src.icon_state))
 	popup.open()
 
+/obj/machinery/computer/libraryconsole/bookmanagement/proc/print_forbidden_lore(mob/user)
+	var/spook = pick("blood", "brass")
+	var/turf/T = get_turf(src)
+	if(spook == "blood")
+		new /obj/item/weapon/tome(T)
+	else
+		new /obj/item/clockwork/slab(T)
+
+	user << "<span class='warning'>Your sanity barely endures the seconds spent in the vault's browsing window. The only thing to remind you of this when you stop browsing is a [spook == "blood" ? "dusty old tome" : "strange metal tablet"] sitting on the desk. You don't really remember printing it.[spook == "brass" ? " And how did it print something made of metal?" : ""]</span>"
+	user.visible_message("[user] stares at the blank screen for a few moments, \his expression frozen in fear. When \he finally awakens from it, \he looks a lot older.", 2)

```

Comment:
BAD! Use [user.p_their()] instead of \his and [user.p_they()] instead of \he (and [user.p_s()] for the s at the end of awakens and looks)
`user.visible_message("[user] stares at the blank screen for a few moments, [user.p_their()] expression frozen in fear. When [user.p_they()] finally awaken[user.p_s()] from it, [user.p_they()] look[user.p_s()] a lot older.", 2)`